### ISSUE TITLE
problem with gris select

### ISSUE THREAD

**Issue by orpiperno** [AntonelloDN/Envimet-inx#6]
Hi AntonelloDN 

I started using your ENVI-met plugin and had trouble setting up the grid. France here the error message. I would love to know what this is and how to solve the problem.

Thank you

Or Piperno
![grid problem](https://user-images.githubusercontent.com/75029623/101278165-512cb680-37c2-11eb-8e46-3dd37daa5f92.JPG)

**Comment by AntonelloDN** [comment]
Hi @orpiperno,

Which version of Sketchup do you use?

Thanks,
Antonello

**Comment by orpiperno** [comment]
Hi Antonello

Now use in 2020 version after I saw in the forum you that it is not work in 2017 version

I would like to get from you help with my model.
After I have inserted a model into SketchUp and given each entity values of ENVIMET layers now I try to save the file as INX and open it in Spaces but it does not let me after saving open the file.
I enclose the sketchup file here:
1. Can I get this help for the second time in total? I put a model into the software and give ENVIMET values.
2. Why can't I open the file after saving?
3. I would love to get a brief explanation of how I define values in the Create ENVI-MET grid. I went through Antuel's video but I still do not understand it.

Thank you for help


Best regards,

Or Piperno
Environmental planning
[cid:image001.jpg@01D6CE01.47B43590]
www.yozmot-sviva.com<http://www.yozmot-sviva.com/>

Address: Nata Yellin Mor St 21, Tel Aviv
Phone: +972-77-3508001
